README.md

```diff
@@ -1,34 +1,30 @@
-# EIPs [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+# Ethereum Improvement Proposals (EIPs) [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/ethereum/EIPs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
 Ethereum Improvement Proposals (EIPs) describe standards for the Ethereum platform, including core protocol specifications, client APIs, and contract standards.
 
-A browsable version of all current and draft EIPs can be found on [the official EIP site](http://eips.ethereum.org/).
+**Before you initiate a pull request**, please read the process document. Ideas should be thoroughly discussed prior to opening a pull request, such as on the [Ethereum Magicians forums](https://ethereum-magicians.org) or in a GitHub issue in this repository.
```

Comment:
The second sentence is now emphasized in EIP-1. Perhaps not needed here.

Reply:
Good call, addressed in 90f04bd